gaston/second-assignment/index.js

```diff
@@ -0,0 +1,34 @@
+let arrHeroes = [];
+
+function getData() {
+  const url = "/marvel.json";
```

Comment:
There's an error loading the file because you need to use a relative path to where's the file is inside the project folder structure.